sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ResourceTraits.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, Text, Tag } from 'grommet';
+
+export const ResourceTraits: React.FC = () => (
+  <Box gap="small">
+    <Text>Traits</Text>
+    <Box direction="row" gap="xsmall">
+      <Tag value="" name="Other" />
```

Comment:
Add `onClick` to these? In their UI design it looks like these would be clickable.

Reply:
added